### PR TITLE
Add badges. Changed install recommendation to conda-forge. Remove some old documentation.

### PR DESCRIPTION
**Pull request by erikmannerfelt**
Badges!!!

The conda-forge install is now up, so this should be the recommended installation procedure.

Also, the README had some outdated text saying "we should get it on readthedocs" (done) and "we need testing" (done, and specified in CONTRIBUTING.md), so I removed those paragraphs. You can see how my suggested frontpage would look here:

https://github.com/erikmannerfelt/GeoUtils/tree/update_readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,35 @@
...
-* Activate environment
 
-`conda activate geoutils`
+## Installation
 
-* Install this package
+#### With conda (recommended)
+First, set a strict channel priority. This seems to be optional (but recommended) on UNIX-based systems, but essential for Windows:
+```bash
+conda config --add channels conda-forge
```
Comment (adehecq) [inline README.md:18]: That is a bit intrusive to ask the users to change their global conda settings. Isn't there a way to set the channel only for this install?

File: README.md [hunk 2]
```diff
@@ -48,9 +57,30 @@ You can find ways to improve the libraries in the [issues](https://github.com/Gl
 Direct pushing to the GlacioHack repository is not permitted.
 
 
-### Documentation - please read ! ###
-In the interest of keeping the documentation simple, please write all docstring in reStructuredText (https://docutils.sourceforge.io/rst.html) format - eventually, we will try to set up auto-documentation using sphinx and readthedocs, and this will help in that task.
+### Development installation
+* Clone the main repo (or your fork)
+```bash
+git clone https://github.com/GlacioHack/GeoUtils  # Or your own fork
```
Comment (adehecq) [inline README.md:63]: They should definitely fork the repository if they want to make any change as they won't be able to push to the main repo. 
Should we have something more detailed, similar to what I wrote for the GlacioHack? https://github.com/GlacioHack/xdem/wiki/Taking-part-to-a-GlacioHack

File: README.md (hunk 1 above)
Comment (erikmannerfelt) [reply README.md:18]: Yes, I just learned. Using `conda install --strict-channel-priority`

File: README.md (hunk 2 above)
Comment (adehecq) [reply README.md:63]: Although that page describes how to contribute to both xdem and geoutils at the same time, so we should have a simpler version for geoutils.

Comment (erikmannerfelt) [reply README.md:63]: I mean they can clone the upstream and add their fork later, that's what I meant! Just for testing, they don't need a fork. It's when they want to contribute that this is needed. 

Maybe simple-turned-overly-complicated from my part? Shall I just say "clone your fork"?

Comment (adehecq) [reply README.md:63]: We should just update the contributing.md file accordingly I guess.

Comment (adehecq) [reply README.md:63]: And reference to it in the main README.

Comment (erikmannerfelt) [reply README.md:63]: Yeah makes sense. The install instructions for contributing should be in CONTRIBUTING.md, not the README haha

Comment (erikmannerfelt) [reply README.md]: There is already a dev-install instruction in CONTRIBUTING.md, so I just referenced to that file in 15fa5df

File: README.md (hunk 1 above)
Comment (erikmannerfelt) [reply README.md]: Replaced this in 15fa5df

**Review by adehecq — APPROVED** [review]

**Pull request merged**